NumbersIntervalES5.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<script>
+    for (var i = 1; i <= 10; i++) {
```

Comment:
Prefer let over var

Reply:
This JS code is done using ES5 features thats why let is not used there as it was mentioned